tests/template_tests/test_engine.py

```diff
@@ -21,6 +22,38 @@ def test_basic_context(self):
         )
 
 
+class GetDefaultTest(SimpleTestCase):
+
+    @override_settings(TEMPLATES=[])
+    def test_raises_when_no_dtl_engines_configured(self):
+        """
+        A configured DTL backend is required.
+        """
+        with self.assertRaisesMessage(ImproperlyConfigured, "No DjangoTemplates backend is configured."):
+            Engine.get_default()
+
+    @override_settings(TEMPLATES=[
+        {
+            'NAME': 'default',
+            'BACKEND': 'django.template.backends.django.DjangoTemplates'
+        },
+        {
+            'NAME': 'extra',
+            'BACKEND': 'django.template.backends.django.DjangoTemplates'
+        },
+    ])
+    def test_multiple_dtl_engines_configured(self):
+        """
+        #27359 -- A DTL backend is returned when several are configured.
+
+        * Users can configure multiple DTL backend (with different options etc).
+        * Third-party libraries still need to be able to call `Template(...)`
+            without passing an explicit engine.
+        """
+        engine = Engine.get_default()
+        self.assertTrue(isinstance(engine, Engine))
```

Comment:
@das-g Hmmm. Not sure. All we're testing there is that `list` is ordered, but we know that... 

The `assertTrue` here isn't really testing very much. The test is really that the `get_default` call doesn't raise an error. 

Bottom line is, I don't really mind.